### DISCUSSION TITLE
Filter Cars

### DISCUSSION THREAD

**Discussion by akac** [teslamate-org/teslamate#918] [Ideas]
**What would you like to be added**:
Select which car shows up first, or to visually filter one of the cars. 

**Why is this needed**:
We have two cars. My wife has no interest in the data and it gets annoying for me to go into Grafana and have to constantly hide her car (hers was first, so it shows up first). 

Ideally I'd like to keep capturing her data, but to make sure that unless I need to see her's -  I'm only seeing my car.

**Additional context**:
When I've tried to hide her car in Grafana it tells me that it can't save the changes because they are provided by Teslamate.

**Comment by tobiasehlert** [comment]
This is pretty similar to the request I've added, which is to totally disable logging for a car.
My issue is https://github.com/adriankumpf/teslamate/issues/1049

**Comment by focher** [comment]
I definitely like this idea. My use case is that we returned one of our Teslas when its lease ended. I wanted to archive or hide it now. I know I can delete it. Other idea is to export the data then erase it.

**Comment by espenu** [comment]
Just giving my support to this idea
